include/logger/loggerConfig.h

```diff
@@ -326,24 +326,49 @@ typedef struct _CANConfig {
 #endif
 } CANConfig;
 
+/* define max offsets and length for CAN mappings */
+#define MAX_CAN_MAPPING_OFFSET_BYTES 7
+#define MAX_CAN_MAPPING_LENGTH_BYTES 4
+
+enum CANMappingEndian {
+    CANMappingEndian_Big = 0,
```

Comment:
I wasn't sure which way to default, but since the processor is little endian by default, I just chose that as the default as well. I think it's about 50/50 out there for CAN mappings in the wild.